chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -11,12 +11,18 @@ metadata:
   name: gateway
   namespace: {{ .Release.Namespace | quote }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.gateway.replicas }}
   template:
     metadata:
       labels:
         app: gateway
     spec:
+      {{- if .Values.operator.create }}
+      serviceAccountName: openfaas-operator
+      {{- end }}
+      securityContext:
+        readOnlyRootFilesystem: true
```

Comment:
Interesting but is it required in this PR? Why haven't you added it to every service where it applies?